tests/test_uploads.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+import sys
+import mock
+from unittest import skip, TestCase
+from unittest.mock import patch, mock_open
+
+from jwplatform import __version__
+from jwplatform.client import JWPlatformClient
+import logging
+
+# from .mock import JWPlatformMock
+from jwplatform.upload import UploadType
+from tests.mock import JWPlatformMock
+
+
+class TestUploads(TestCase):
+
+    @mock.patch("os.stat")
+    def test_upload_method_is_direct_when_file_size_is_small(self, os_stat):
+        file_content_mock = 'some bytes'
+        os_stat.return_value.st_size = 4 * 1024 * 1024
+        with patch("builtins.open", mock_open(read_data="data")) as mock_file:
```

Comment:
This could probably be moved up to where the `os.stat` patch is

Reply:
Fixed.